components/proxy/src/main/java/com/hotels/styx/infrastructure/logging/LOGBackConfigurer.java

```diff
@@ -121,26 +120,6 @@ public static void initLogging(URL url, boolean installJULBridge) {
         }
     }
 
-    /**
-     * Shut down LOGBack.
-     * This isn't strictly necessary, but recommended for shutting down
-     * logback in a scenario where the host VM stays alive (for example, when
-     * shutting down an application in a J2EE environment).
-     *
-     * @param uninstallJULBridge should an attempt be made to uninstall the JUL bridge
-     */
-    public static void shutdownLogging(boolean uninstallJULBridge) {
-        ContextSelector selector = ContextSelectorStaticBinder.getSingleton().getContextSelector();
-        LoggerContext loggerContext = selector.getLoggerContext();
-        String loggerContextName = loggerContext.getName();
-        LoggerContext context = selector.detachLoggerContext(loggerContextName);
-        if (uninstallJULBridge) {
-            SLF4JBridgeHandler.uninstall();
-        }
-        context.stop();
-    }
-
-
```

Comment:
Due to logback internal changes this needed to be rewritten as it wasn't compiling anymore. I've opted in removing this as we are not deploying styx in J2EE and it didn't worth investing effort in it.